src/os2datascanner/projects/admin/tests/test_miniscanner.py

```diff
@@ -0,0 +1,135 @@
+from os2datascanner.projects.admin.adminapp.views.miniscanner_views import *
+from random import choice
+
+class DummyFile():
```

Comment:
I believe this class is duplicative of `django.core.files.uploadedfile.SimpleUploadedFile`; you should probably just use that one.

Reply:
Let me check if it works. If so, i'll change it.

Reply:
Indeed, it works. Thanks !